packages/test/index.js

```diff
@@ -10,7 +10,7 @@ let rtrace = new Rtrace({
 });
 const fs = require("fs");
 
-const mod = loader.instantiateSync(fs.readFileSync("./test/index.wasm"), rtrace.install({
+const mod = loader.instantiateSync(fs.readFileSync("./build/debug/index.wasm"), rtrace.install({
```

Comment:
I know using asbuild is well intentioned. I would prefer to use the compiler directly. Please revert this change.